websites/Z/Zee5/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "993170607841746974",
+		"name": "பில்லி கசாப்புக்காரன்"
+	},
+	"service": "Zee5",
+	"description": {
+		"en": "Zee5 is an OTT platform"
```

Taken from the website
```suggestion
		"en": "ZEE5 is the largest streaming platform where you can watch your favorite TV show before it is aired on ZEE5. Stream blockbuster movies, latest originals, music videos, live TV channels and news in HD quality & in a language of your choice."
```